packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2507,6 +2534,7 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
       return new _CleanSemanticsFragment(renderObjectOwner: this);
     }
     List<_SemanticsFragment> children;
+    bool dropSemanticsOfLeftSiblings = (this is RenderBlockSemantics);
```

Comment:
Usually its a better pattern to call a method on `this` rather than testing `this` against a specific subclass.